### ISSUE TITLE
Repo transfer

### ISSUE THREAD

**Issue by mxmCherry** [mxmCherry/openrtb#54]
Can https://github.com/prebid take over this repo? Fork under the org, change paths and maintain it (usually it's a low-effort activity). Maybe tag it as v1 after fork (reset versioning after it gains new URL).

CC @SyntaxNode (as the latest contributor related to Prebid) for help / contact.

I'm not interested in maintaining it - I'm neither using it nor enjoying following RTB standards anymore.

I'd prefer to point to / recommend active fork (forked by active lib user) before archiving it under my account.

**Comment by SyntaxNode** [comment]
Makes sense. We still use actively use it for Prebid Server and continuing the legacy makes sense to me. Let me ask the Prebid folks at our next meeting on September 16.

**Comment by SyntaxNode** [comment]
Prebid accepts the proposal to take over this project. Do you have any issue with us changing the fork to the Apace 2.0 license? That's the standard license Prebid uses in it's open source projects.

**Comment by mxmCherry** [comment]
No problem at all, go ahead - I have no issues with using Apache 2.0.

**Comment by SyntaxNode** [comment]
Great. Thank you. I've received all necessary approvals on the Prebid side. I plan to have the fork ready this week.

**Comment by SyntaxNode** [comment]
I think we're all set. There was already a Prebid fork of this library at version 3.0.0 from ~4 years ago. Not sure why, but it prevents me from resetting the version to v1 as Go module discovery was still seeing 3.0.0 as the latest. I decided to keep with the v17 version and move on from here, as calling it v4 seemed unnecessarily confusing. 

https://pkg.go.dev/github.com/prebid/openrtb/v17 looks good. I've opened a PR in Prebid Server to verify it works outside of my machine.
